doc/flux_to_redux.md

```diff
@@ -0,0 +1,579 @@
+        }
+
+        return oldState;
+    }
+    ```
+1. __Create the state model.__ After pulling out state mutation logic, this should be as simple as renaming the class, not inheriting from the Flux store class, and cleaning up anything unrelated to the actual state fields. 
+1. __Instantiate the store class, using the state model and reducer.__
+1. __Add a `ReduxProvider` around the component tree.__ This will take in the store instantiated in step 4 as the component's `store` prop.
+1. __Refactor componentry to not longer be a `FluxUiComponent` and instead be a connected `UiComponent2`.__ In terms of moving away from Flux, the simplest case is that props and prop calls need to be updated. Props will need to be added to make room for the ones consumed by the Redux component, and any `props.actions` calls need to be updated to `props.dispatch` (unless you're using `mapDispatchToProps`). 
+
+## An Advanced Example
+In the case a Flux app has a complex store architecture that involves multiple stores, whether they be nested under one store or be completely separate, the transition process has a few additional catches. 
+
+This section builds on the [simple example](#a-simple-example), so ensure that section makes sense conceptually before looking at multiple stores.
+
+### An Adanced Flux App
+- __The Flux Store Classes:__ `RandomColorStore`, `MidLevelStore`, `LowLevelStore`, and `AnotherColorStore`. `RandomCOlorStore`, `MidLevelStore`, and `LowLevelStore` are nested within each other while `AnotherColorStore` is completely separate.
+
+    Despite there being four stores, all of them have the same job: handle a single random color property. Therefore, they are all nearly identical objects. Naturally in the real world they would likely be drastically different, but for simplicity sake and keeping the focus on the big picture they are all left the same.
+- __The Store Objects:__ `bigStore` and `littleStore`
+- __The Actions Class__, `RandomColorActions`
+- __The Actions Object__, `randomColorActions`
+- __The UI:__ The only complex component is a `BigBlock` component that receives `randomColorActions`, `bigStore`, and `littleStore` as props. Updates are triggered by using the action prop. Naturally now state is accessed via:
+    - props.bigStore.state
+    - props.bigStore.midLevelStore.state
+    - props.bigStore.midLevelStore.lowLevelStore.state
+    - props.littlebig_Store.state
+
+    Our `BigBlock` component renders three `SmallBlock` components. Each one connects to a different store's state, with the remaining background color being tied to `BigBlock` itself. 
+    
+    `BigBlock` also has four buttons - one connected to each action that changes the background color property of one of the stores (and thereby updating the component background color).
+
+    A Flux specific part of the `BigBlock` component is that we need to set the `redrawOne` to listen to the story at every level.
+
+```dart
+// A nested store
```

Comment:
I think we should show the necessary imports for clarity as to what the `flux` namespace represents.

```suggestion
import 'package:w_flux/w_flux.dart' as flux;
    
// A nested store
```